src/core/CCOpt.ml

```diff
@@ -54,10 +58,16 @@ let (<*>) f x = match f, x with
 
 let (<$>) = map
 
-let (<+>) a b = match a with
+let or_ b a = match a with
```

Comment:
It also seems natural to use it as `a |> or_ b` (though preferably in a longer chain)